omero/users/cli/import.txt

```diff
@@ -22,22 +32,22 @@ Some of the options available to the import command are:
 
 .. option:: -h, --help
 
-    Display the help for the import command
-
-.. option:: --java-help
-
-    Display the help for the Java options of the import command
+    Display the help for the import command, use::
 
-    Java options can be passed after ``--``, e.g.::
+        $ bin/omero import -s SERVER -p PORT -u USER -g GROUP image.tif
 
-       bin/omero import image.tif -- --name=test --description=description
+.. option:: -d DATASET_ID, -T TARGET, --target TARGET
+    
+    To import images into a Dataset ::
 
-.. option:: --advanced-help
+        $ bin/omero import image.tif -d 2  
+        $ bin/omero import image.tif -T Dataset:id:2
+        $ bin/omero import image.tif -T Dataset:name:Sample01
 
-    Display the advanced help for the import command
+    See :doc:`import-target` for more information on import targets.
 
-    Some advanced import options are described in the :ref:`inplace_import`
-    section.
```

Comment:
Fine to remove this, but maybe add a link to the section you have added to this page below called ``Advanced import commands``. You can do it using the :ref: logic and mentioning ``See Advanced... below``